omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -376,8 +383,14 @@ private boolean isSameUser(PageRequest pageRequest, String username) {
 	}
 	
 	private List<Location> getUserLocations(AdministrationService adminService, LocationService locationService) {
-		String locationUserPropertyName = adminService
-		        .getGlobalProperty(ReferenceApplicationConstants.LOCATION_USER_PROPERTY_NAME);
+		String locationUserPropertyName;
+		try {
+			Context.addProxyPrivilege(PrivilegeConstants.GET_GLOBAL_PROPERTIES);
```

Comment:
Does this mean that line 383 below results into a NPE for `Context.getAuthenticatedUser().getUserProperty()`?

Reply:
@dkayiwa I updated the PR.